### PR TITLE
New FundAccount + FundModel

### PR DESCRIPTION
**Pull request by 0x0ece**

### CODE REVIEW

File: anchor/programs/glam/src/state/model/model.rs [hunk 1]
```diff
@@ -87,18 +87,18 @@ pub struct ShareClassOpenfundsModel {
     // Core
     // pub applied_subscription_fee_in_favour_of_distributor: Option<String>,
```
Comment (yurushao) [inline anchor/programs/glam/src/state/model/model.rs:88]: Why lots of fields are being commented out?

File: anchor/src/client.ts [hunk 2]
```diff
@@ -180,28 +189,44 @@ export class GlamClient {
...
+            manager,
+            tokenProgram: TOKEN_2022_PROGRAM_ID
+          })
+          .preInstructions([
+            ComputeBudgetProgram.setComputeUnitLimit({ units: 500_000 })
+          ])
+          .rpc();
+      })
+    );
     return [txSig, fundPDA];
```
Comment (yurushao) [inline anchor/src/client.ts:230]: The method is typed by `Promise<[TransactionSignature, PublicKey]>` but return type doesn't match that.

Comment (0x0ece) [reply anchor/src/client.ts:230]: ?
async is promise

File: anchor/programs/glam/src/state/model/model.rs (hunk 1 above)
Comment (0x0ece) [reply anchor/programs/glam/src/state/model/model.rs:88]: not implemented / too many fields = too costly to deser :(

File: anchor/programs/glam/src/state/model/openfunds.rs [hunk 3]
```diff
@@ -75,6 +75,25 @@ impl From<FundModel> for Vec<FundField> {
 impl From<&ShareClassModel> for Vec<ShareClassField> {
     fn from(model: &ShareClassModel) -> Self {
         let mut res = vec![];
+        // Derived fields
+        let is_raw_openfunds = model.is_raw_openfunds.unwrap_or(false);
+        let model = model.clone();
+        if !is_raw_openfunds {
+            //TODO
```
Comment (yurushao) [inline anchor/programs/glam/src/state/model/openfunds.rs:82]: Can you document the TODO if it's still needed?

**Review by yurushao — APPROVED** [review]

**Pull request merged**